### PR TITLE
Pre-set localization labels on issues raised from MDN content

### PR DESCRIPTION
**Pull request by dontcallmedom**
Similarly as done for content labels

### CODE REVIEW

**Comment by peterbe** [comment]
Would you mind merge in https://github.com/dontcallmedom/yari/pull/1 (if you approve) and also merge in latest `main` so we can fully test it.

File: client/src/document/on-github.tsx [hunk 1]
```diff
@@ -112,6 +120,9 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
   sp.set("title", `Issue with "${titleShort}": (short summary here please)`);
 
   const slug = doc.mdn_url.split("/docs/")[1].toLowerCase();
+  const { locale } = doc;
+
+  let labels = ["needs-triage"];
```
Comment (peterbe) [inline client/src/document/on-github.tsx:125]:
```suggestion
  const labels = ["needs-triage"];
```

Comment (peterbe) [reply client/src/document/on-github.tsx:125]: for consistency and stuff.

**Review by peterbe — COMMENTED** [review]
The code is fine but there's potentially a bigger problem. In particular, line 101 :)

If I'm viewing a French document and I spot something incorrect, shouldn't I be using https://github.com/mdn/translated-content/issues/new?labels=l10n-fr instead?

And over in that repo, they don't use `Content:...` by the way. 

I'm thinking perhaps it can be something like this:

```js
const {locale} = doc;
const labels = ['needs-triage'];
if (locale.toLowerCase() === 'en-us') {
  labels.push(`Content:${prefix}`)
} else {
  labels.push(`l10n:${localeAlias}`)
}
```

**Comment by dontcallmedom** [comment]
ha! excellent remark indeed; I'll update the patch to point to translated-content instead

**Review by peterbe — APPROVED** [review]
Works perfectly!

**Pull request merged**